.eslintrc.js

```diff
@@ -143,6 +143,9 @@ module.exports = {
         ],
       },
     ],
+    // https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unused-vars.md
+    'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': ['error'],
```

Comment:
I believe this can also be fixed by switching the order of
```js
extends: [
  'plugin:@typescript-eslint/recommended',
  'eslint:recommended',
  // ...
]
```
to
```js
extends: [
  'eslint:recommended',
  'plugin:@typescript-eslint/recommended',
  // ...
]
```
in the ESLint conf.

Reply:
Good call. Done in https://github.com/WordPress/openverse-frontend/pull/1129/commits/38ee96867d8dee5856a1b43702402e30742aafb2